packages/devtools_app/lib/src/ui/utils.dart

```diff
@@ -288,9 +286,7 @@ class ScreenSize {
     if (height < 450) return MediaSize.xs;
     if (height < 600) return MediaSize.s;
     if (height < 750) return MediaSize.m;
-    if (height < 900)
-      return MediaSize.l;
-    else
-      return MediaSize.xl;
+    if (height < 900) return MediaSize.l;
```

Comment:
Fyi @pq this sort of case is probably one we would want to start suggesting and quick fixing to Patterns.

Reply:
See: https://github.com/dart-lang/linter/issues/3673